user/pages/30.blog/2018-09-05_experiments-in-water-recycling/item.en.md

```diff
@@ -26,10 +26,10 @@ We found out that washing machine waste water is rich in nutrients and low in ox
 This is an ideal environment for the growth of anaerobic bacteria, which like to produce smelly gas.
 As we did never empty the tank in the basement completely, sludge did accumulate at the bottom, making it impossible to pump it out.
 
-![](tank_sludge.jpg)
+![](tank_sludge.jpg)<br />
```

Comment:
`br` is self-closing, i.e. writing `<br>` is correct: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br